sdk/mobile-center-crashes/src/main/java/com/microsoft/azure/mobile/crashes/Crashes.java

```diff
@@ -142,6 +166,10 @@ private Crashes() {
         mCrashesListener = DEFAULT_ERROR_REPORTING_LISTENER;
         mUnprocessedErrorReports = new LinkedHashMap<>();
         mErrorReportCache = new LinkedHashMap<>();
+        HandlerThread thread = new HandlerThread(THREAD_NAME);
```

Comment:
Make sure last session error report is loaded first, then process pending crashes.
Then quit looper?